### PR TITLE
Use ffmpeg sys

### PR DESCRIPTION
**Pull request by MarcAntoine-Arnaud**

### CODE REVIEW

File: src/prelude.rs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  tools,
+  video_decoder::VideoDecoder,
+  video_encoder::VideoEncoder,
+};
+
+#[doc(no_inline)]
+pub use ffmpeg_sys::*;
+
+#[doc(no_inline)]
+pub use ffmpeg_sys::*;
```
Comment (Romanelaf) [inline src/prelude.rs:33]: why is it declared 2 times ?

File: Cargo.toml [hunk 2]
```diff
@@ -13,13 +13,15 @@ license = "Apache-2.0"
 repository = "https://github.com/nomalab/stainless-ffmpeg"
 
 [dependencies]
-stainless-ffmpeg-sys = "4.2.3"
+ffmpeg-sys = "4.2.1"
```
Comment (Romanelaf) [inline Cargo.toml:16]: oh nice, I hadn't seen the crate has been updated

**Review by Romanelaf — APPROVED** [review]

File: src/prelude.rs (hunk 1 above)
Comment (MarcAntoine-Arnaud) [reply src/prelude.rs:33]: true ...

**Review by Romanelaf — APPROVED** [review]

**Comment by MarcAntoine-Arnaud** [comment]
Thank you @Romanelaf !

**Pull request merged**